### PR TITLE
Final V1.0 Specification

### PR DESCRIPTION
**Pull request by apinilabs-daniel**
Please review the Fina V1.0 Specification of the `PAC-ID Resolver`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,104 +1,168 @@
-# PAC-ID Resolver (DRAFT)
+# PAC-ID Resolver
+
+## What Is a `PAC-ID Resolver`?
+
+> [!TIP]
+> A `PAC-ID Resolver`, short for **P**ublicly **A**ddressable **C**ontent **ID**entifier **Resolver**, is a program, service or library designed to convert a given `PAC-ID` into a URL. This URL can then be utilized to access additional information associated with the corresponding `PAC-ID`.
```
Comment (apinilabs-pascal) [inline README.md]: link to PAC-ID specification

File: README.md [hunk 2]
```diff
@@ -1,104 +1,168 @@
...
+
+> [!NOTE]
+> Example of a mapping table:
+> ```
+> # mapping table version: 1.0
+> # This is a comment
+> Service Name        ⇨ User Intent ⇨ Service Type         ⇨ Applicable if                       ⇨ Template Url
+> Product Information ⇨ ProdInfo    ⇨ userhandover-generic ⇨ {isu}=METTORIUS.COM;{idSeg1}=DEVICE ⇨ https://www.mettorius.com/inventory/{idSeg1}/{idVal21}
+> Attributes          ⇨ Attributes  ⇨ attributes-generic   ⇨ {isu}=METTORIUS.COM                 ⇨ https://attributes.mettorius.com/{id}
+> # ... more entries ...`
```
Comment (apinilabs-pascal) [inline README.md]: ` can be removed

File: README.md [hunk 3]
```diff
@@ -1,104 +1,168 @@
...
 
-**Q: What if a mapping URL is changing?**
+#### Rules
 
-**A:** Update the mapping table. Use a local override, like a user or corporate mapping table if needed.
+In the context of this specification, a `rule` consists of a `variable` followed by an `=` sign and a `value`, serving for comparison against a specific value the `variable` refers to inside the `PAC-ID`. Alternatively, if a value for the `variable` merely needs to exist without specifying an exact value, the `variable` can stand alone without an assigned value.
 
-**Q: How to use only a part of the id or category in placeholders?**
+> [!NOTE]
+> Example of a `rule` matching `PAC-ID`s with `issuer` "METTORIUS.COM" and having any non-empty `id segment value` available for the `id segment key` "21":
```
Comment (apinilabs-pascal) [inline README.md]: This is a check of key existence not of the value being non-empty of that key. at least not explicitly

File: README.md [hunk 4]
```diff
@@ -1,104 +1,122 @@
...
+A `PAC-ID Resolver`, short for **P**ublicly **A**ddressable **C**ontent **ID**entifier **Resolver**, is an architectural pattern for loosely coupling applications, creating modern user experience without classical integration. The basis for the coupling is a `PAC-ID`. 
 
-The PAC-ID resolver architecture introduced here offers an alternative and lightweight solution for implementing handover between applications and devices in a loosely coupled manner, providing an exceptional user experience.
+Working principle: The application asks the `PAC-ID Resolver` to convert a given `PAC-ID` into an ordered list of coupling information, based on configurable coupling information tables provided to the `PAC-ID Resolver`. The application in turn selects the best coupling information from the list, based on an application intent. The coupling information contains a URL, which can be used by the application to either handover the user to another application or to find relevant end-points for inter application communication.
```
Comment (apinilabs-manuel) [inline README.md]: The coupling information, depending on the indicated service type, contains a URL (e.g. a deep link to a target system) for user handovers or a endpoint for standardized inter-application communication.

(The original wording indicates that the URL is for finding relevant endpoints but in reality the url itself is the endpoint.

File: README.md [hunk 5]
```diff
@@ -1,104 +1,122 @@
...
 
-The PAC-ID resolver architecture has been designed with the following design goals:
-- **Decentralized operation**: The PAC-ID resolver architecture should be able to operate independently without relying on a central registry.
-- **Local customization**: It should allow local overrides of PAC-ID to link conversion mechanisms, enabling customization for local context specific needs.
-- **Flexibility and independence**: The PAC-ID resolver architecture should function without strict governance, providing flexibility and independence in its operation.
-- **Isolated network compatibility**: It should be capable of operating effectively in isolated networks, even with limited or no connectivity to external resources.
+The PAC-ID Resolver architecture offers an alternative and lightweight solution for connecting applications in a loosely coupled manner, depending on the current workflow context. This architecture provides an exceptional user experience by facilitating seamless interactions between applications.
```
Comment (apinilabs-manuel) [inline README.md]: In contrast, the PAC-ID Resolver architecture offers an alternative and lightweight solution for connecting applications in a loosely coupled manner. Implemented as a neutral contract between systems, it eliminates inter-dependencies and therefore facilitates adding additional or replacing existing systems. Its workflow context awareness allows seamless interactions that result in an exceptional user experience.

File: README.md [hunk 6]
```diff
@@ -1,104 +1,122 @@
...
 
-## Specification
-A PAC-ID resolver is a service or library used by a source application to resolve a PAC-ID into one or more browsable link(s). 
+## Interaction of an Application with a `PAC-ID Resolver`
```
Comment (apinilabs-manuel) [inline README.md:17]: Coupling flow? Application coupling sequence?

**Pull request merged**